cmd/gateway/main.go

```diff
@@ -26,13 +25,14 @@ var (
 	gatewayCtlrName = flag.String(
 		"gateway-ctlr-name",
 		"",
-		fmt.Sprintf("The name of the Gateway controller. The controller name must be of the form: DOMAIN/NAMESPACE/NAME. The controller's domain is '%s'; the namespace is '%s'", domain, namespace),
+		fmt.Sprintf("The name of the Gateway controller. The controller name must of the form DOMAIN/PATH. The controller's domain is '%s'", domain),
```

Comment:
"The controller name must of the "

We need a verb to replace "be".

Reply:
Fixed